Cargo.toml

```diff
@@ -12,18 +12,18 @@ license = "MIT OR Apache-2.0"
 categories = ["config"]
 
 [features]
-env = ["pear", "parse-value"]
-json = ["serde_json"]
-yaml = ["serde_yaml"]
-parse-value = ["pear"]
-test = ["tempfile", "parking_lot"]
-# toml = ["toml"]
```

Comment:
This raises the MSRV to 1.60. This could simply be `toml = ["toml_edit"]` without impacting the MSRV.